Frameworks/IntelPython_XGBoost/IntelPython_XGBoost_GettingStarted/README.md

```diff
@@ -0,0 +1,105 @@
+# Intel Python XGBoost Getting Started
```

Comment:
Old styleReadme, please update using the [mandelbrot readme](https://github.com/oneapi-src/oneAPI-samples/blob/master/DirectProgramming/DPC%2B%2B/CombinationalLogic/mandelbrot/README.md) as an example: In this case a "Purpose" and "License" sections are missing at a minimum